OSDOCS 12042: Added release note update for supporting partition placement group in AWS (#82436)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

[id="ocp-4-17-machine-management_{context}"]
=== Machine management

[id="ocp-4-17-enhancements-aws-mapi-placement_{context}"]
==== Supporting AWS Placement Group Partition Number

> This release introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].

Reviewer: Consider using the attribute for AWS: 

![Screenshot from 2024-10-03 13-37-19](https://github.com/user-attachments/assets/6076bd59-bf24-4abf-b880-fccc81a68130)

> This release introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].

Reviewer: ```suggestion
This release introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. With this feature, you can specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```

"Allows" is a form of anthrophomorphism.

[id="ocp-4-17-machine-management_{context}"]
=== Machine management

[id="ocp-4-17-enhancements-aws-mapi-placement_{context}"]

> ==== Supporting AWS Placement Group Partition Number in CAPI & MAPI

Reviewer 2: I don't think we need to explicitly call out CAPI & MAPI in RN
```suggestion
==== Supporting AWS Placement Group Partition Number
```

> The {product-title} release {product-version} introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].

Reviewer 3: The beginning here can be simplified since it's going right in the 4.17 RNs:
```suggestion
This release introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```

> The {product-title} release {product-version} introduces the `placementGroupPartition` field for {product-title} `MachineSets` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For more information, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].

Reviewer 2: - `MachineSet` is the correct term. https://github.com/openshift/api/blob/3c700b6cb32b58a619428e246b4ad5040565ac13/machine/v1beta1/types_machineset.go#L28

```suggestion
The {product-title} release {product-version} introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```
